zkdb/src/api/apollo/validation.ts

```diff
@@ -32,6 +32,14 @@ export const validateCID = Joi.string()
   .pattern(/^b[a-z,2,3,4,5,6,7]+(|=)+$/i)
   .message('Invalid CID');
 
+export const validateIndex = Joi.string()
```

Comment:
Can we cast the value as `number`? like `joi.integer()` so the validation will be cheaper since it don't need Regex.

Reply:
For index we use BigInt, does the integer will be enough or try to use BigInt?